src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -29,6 +29,7 @@ protocol CoronaWarnAppDelegate: AnyObject {
 	var exposureManager: ExposureManager { get }
 	var taskScheduler: ENATaskScheduler { get }
 	var serverEnvironment: ServerEnvironment { get }
+	var warnOthers: WarnOthersRemindable { get }
```

Comment:
This instance should then also be called `warnOthersReminder`, I think

Reply:
Renamed... 👍